### ISSUE TITLE
[CRASH] when entering r/burlington

### ISSUE THREAD

**Issue by dpreschel** [lo-cafe/winston#189]
**Describe the bug**
App crashes and closes when attempting to enter a specific sub, but not all.  

**To Reproduce**
1. Go to sub list menu (select ‘Burlington’ if subscribed),  or search for sub. ‘Burlington’
2. App attempts to open sub but crashes

**Expected behavior**
Sub should open and see posts

**Screenshots**
If applicable, add screenshots or screen recordings to help explain your problem.

**Desktop (please complete the following information):**
 - OS: ios 17.0.3
 - Version beta 1.0
 - Build Number 144

**Device (please complete the following 
information):**
 - Device: iPhone 14,  iPad 10th gen
 - OS: iOS 17.0.3

**Additional context**

**Comment by dpreschel** [comment]
Tried it today and they worked… both r/burlington and r/obsidian do now don’t crash the app on same devices.

**Comment by ojsef39** [comment]
> Tried it today and they worked… both r/burlington and r/obsidianmd now don’t crash the app on same devices.

i would guess that it wasn't the subreddit itself what crashed the app, maybe it was a post that was displayed in the subreddit and now that post isn't displayed anymore

**Comment by dpreschel** [comment]
Yes most likely a post or something in a post.   It started happening again this morning for me in /r/burlington
